src/hooks/mod.rs

```diff
@@ -0,0 +1,126 @@
+use anyhow::Error;
```

Comment:
per https://doc.rust-lang.org/reference/items/modules.html#module-source-filenames, I think this should be called `src/hooks.rs`. ([stackoverflow thread](https://stackoverflow.com/a/68972967/4972) on the same)

> Note: Prior to rustc 1.30, using mod.rs files was the way to load a module with nested children. It is encouraged to use the new naming convention as it is more consistent, and avoids having many files named mod.rs within a project.

Reply:
Done